build_tools.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
I can break this into multiple steps in the `Dockerfile`, but the build dependencies will have to persist (and thus be distributed with the image) across layers, unless I install and purge build deps at every `RUN` step. Do you have a preference?